### DISCUSSION TITLE
Tokio v1.15.0

### DISCUSSION THREAD

**Discussion by carllerche** [tokio-rs/tokio#4322] [Announcements]
### Fixed

- io: add cooperative yielding support to `io::empty()` ([#4300])
- time: make timeout robust against budget-depleting tasks ([#4314])

### Changed

- update minimum supported Rust version to 1.46.

### Added

- time: add `Interval::reset()` ([#4248])
- io: add explicit lifetimes to `AsyncFdReadyGuard` ([#4267])
- process: add `Command::as_std()` ([#4295])

### Added (unstable)

- tracing: instrument `tokio::sync` types ([#4302])

[#4302]: https://github.com/tokio-rs/tokio/pull/4302
[#4300]: https://github.com/tokio-rs/tokio/pull/4300
[#4295]: https://github.com/tokio-rs/tokio/pull/4295
[#4267]: https://github.com/tokio-rs/tokio/pull/4267
[#4248]: https://github.com/tokio-rs/tokio/pull/4248
[#4314]: https://github.com/tokio-rs/tokio/pull/4314

<hr /><em>This discussion was created from the release <a href='https://github.com/tokio-rs/tokio/releases/tag/tokio-1.15.0'>Tokio v1.15.0</a>.</em>

**Comment by carllerche** [comment]
I'm trying something new. Github now supports associating a release with a discussion.

**Comment by ISibboI** [comment]
Just out of interest: how did the MSRV increase happen?

**Reply by carllerche** [reply]
Happened here: https://github.com/tokio-rs/tokio/pull/4254. It's in preparation to upgrade to Mio 0.8 which requires this version. Tokio 1.0 will require a rustic version no more than ~6 mo old.
